specs/glossary.md

```diff
@@ -634,7 +634,7 @@ See the [Engine Queue section][engine-queue] of the L2 chain derivatiaon spec fo
 The finalized L2 head is the highest L2 block that can be derived from *[finalized][finality]* L1 blocks — i.e. L1
 blocks older than two L1 epochs (64 L1 [time slots][time-slot]).
 
-[finality]: <https://hackmd.io/@prysmaticlabs/finality> *finalized* L1 data.
```

Comment:
I'm not sure what the intent of the extra content after the link was, but it broke the link. You can see that the rich diff fixes it.